js_dev/ibm704sim.js

```diff
@@ -967,6 +967,21 @@ function ADD(computer, address) {
     computer.accumulator.fixed_point += computer.general_memory[address].fixed_point;
 }
 
+/**
+ * Emulates the IMB 704 SUB operation.
+ *
+ * Subtracts the value of the storage register to the accumulator as if it were a
+ * fixed point number.
+ *
+ * @param computer
```

Comment:
Be sure to provide little descriptions for the parameters like in the other doc comments.